### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by brunobertoldi**

### CODE REVIEW

File: json_client.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+import struct
+import logging
+
+logger = logging.getLogger("jsonSocket")
+logger.setLevel(logging.DEBUG)
+FORMAT = '[%(asctime)-15s][%(levelname)s][%(funcName)s] %(message)s'
+logging.basicConfig(format=FORMAT)
+
+
+class JsonSocket(object):
```
Comment (andrelcfernandes) [inline json_client.py]: python 2 feelings

File: json_client.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    def __init__(self, address='127.0.0.1', port=5000):
```
Comment (andrelcfernandes) [inline json_client.py]: documenar q eh pra mudar o endereco pro endpoint

**Comment by andrelcfernandes** [comment]
mudar para cliente async = websocket demo

**Comment by RoGryza** [comment]
There are these minor tweaks suggested by @andrelcfernandes. I've assigned both @HenriqueNogara and @brunobertoldi, whoever can pick this up please do

**Pull request merged**